src/api-v1/paths/transactions/{transaction_id}.ts

```diff
@@ -9,19 +9,19 @@ import { finishRes } from "../../../finishRes";
 export default function(api: RippleAPI, log: Function): Operations {
 
   // Simply logs result of validation to debug output
-  const validate = (res: ValidatableResponse, response: object): void => {
-    // TODO: validate all responses
-    if (process.env.NODE_ENV != 'production') {
-      const validation = res.validateResponse(200, response);
-      if (validation) {
-        // red
-        log('\x1b[31m%s\x1b[0m', '/accounts/{address}/info: validation:', validation);
-      } else {
-        // green
-        log('\x1b[32m%s\x1b[0m', '/accounts/{address}/info: response validated');
-      }
-    }
-  };
+  // const validate = (res: ValidatableResponse, response: object): void => {
```

Comment:
Why are we commenting this?

Reply:
adding a comment, it wasn't being used anywhere in the file so TS now throws "unused" warnings